src/pages/Tools/AddressConversion/ScriptToHash.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useMemo, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { utils } from '@ckb-lumos/base'
+import { RadioGroup, RadioGroupItem } from '../../../components/ui/RadioGroup'
+import { HashType } from '../../../constants/common'
+import styles from './styles.module.scss'
+import CopyableText from '../../../components/CopyableText'
+
+export const ScriptToHash: React.FC = () => {
+  const [codeHash, setCodeHash] = useState('')
+  const [args, setArgs] = useState('')
+  const [hash, setHash] = useState<string | undefined>(undefined)
+  const [hashType, setHashType] = useState(HashType.TYPE)
+  const { t } = useTranslation()
+
+  useMemo(() => {
+    try {
```

Comment:
Use `debounce` so the calculation is only done when the input is finished

Ref:
- debounce: https://www.freecodecamp.org/news/javascript-debounce-example/